Remove vserver (#74)
lxylxy123456/ngshare: README.md

	 as `ngshare`. So the current version of `vserver` should conform to the API
	 documentation at Git version
	 [`890c4b21`](https://github.com/lxylxy123456/ngshare/blob/890c4b2187acc6f592a63b8df9db003226ce2b1e/api-specifications.md).

> 	* May 9, 2020: As adviced in #71, we are removing vserver from the ngshare

Reviewer: ```suggestion
	* May 9, 2020: As advised in #71, we are removing vserver from this repository
```

Author: Maybe "the ngshare repository" is better?
